usecases/objects/authorization_test.go

```diff
@@ -76,12 +76,6 @@ func Test_Kinds_Authorization(t *testing.T) {
 			expectedVerb:     "update",
 			expectedResource: "objects/class/foo",
 		},
-		{
```

Comment:
Why this test case for UPDATE `objects/class/foo` has been removed? it's no longer valid?

Reply:
This test case has been removed because it is redundant. 
If you check the file you will find the test case for UpdateObject.